digdag-standards/src/main/java/io/digdag/standards/operator/jdbc/JdbcConnection.java

```diff
@@ -33,5 +33,8 @@ default String escapeTableReference(TableReference ref)
 
     String escapeIdent(String ident);
 
+    void setDebug(Boolean debug);
+    Boolean getDebug();
+
```

Comment:
This change may break compatibility. For ex. Does `digdag-plugin-mysql` work new version?

Reply:
I am also afraid of breaking compatibility.
How about define as default methods as follows.
```
    default void setDebug(boolean debug) { throw new RuntimeException("..."); }

    default boolean getDebug() { return false; }
```

I don't know we can avoid breaking compatibility by default method.
Could you check the binary with digdag-plugin-mysql works or not?

Reply:
defined default method.